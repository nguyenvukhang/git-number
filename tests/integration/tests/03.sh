# staged file (number)

init 4
_gitnu status
_gitnu add 2 # use number
log gitnu status
