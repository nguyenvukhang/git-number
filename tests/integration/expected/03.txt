On branch main

No commits yet

Changes to be committed:
1	[32mnew file:   file_2[m

Untracked files:
2	[31mfile_1[m
3	[31mfile_3[m
4	[31mfile_4[m

